configuration.yaml

```diff
@@ -1,16 +1,30 @@
 backend: local
-backup_directory: /home/INGInious/backup
-local-config: {}
+backup_directory: /var/www/INGInious/backup
+
+local-config:
+    tmp_dir: /tmp
+
+smtp:
+    sendername: UNcode@unal.edu.co
+    host: smtp.gmail.com
+    port: 587
+    username: nlarranagac@unal.edu.co
+    password: vWMcL8CUg
```

Comment:
Is it your password? I think you should remove it and the email as well, we cannot publish them.